test/views/cancellation/CancelDeclarationViewSpec.scala

```diff
@@ -53,7 +53,7 @@ class CancelDeclarationViewSpec extends AppViewSpec with CommonMessages {
         )
     )
 
-    checkErrorsSummary(view)
+    view.getElementById("error-summary-heading").text() mustNot be(empty)
```

Comment:
Is it enough to just test that it is not empty? It confuses me always on view tests, to what extent do we want our frontend to be tested?